plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/MongoServer.java

```diff
@@ -42,6 +50,11 @@ public ConnectionString getConnectionString()
         return new ConnectionString(dockerContainer.getReplicaSetUrl());
     }
 
+    public ConnectionString getConnectionString(String user, String password)
```

Comment:
Can we create a client directly instead of creating `ConnectionString` and creating based on it.

Reply:
It's also used to provide value for `mongodb.connection-url` property.

Reply:
I see `mongodb.credentials` is marked as deprecated and in my case using `mongodb.credentials` does not work in my test case.

Reply:
Both `mongodb.credentials` and `mongodb.connection-url` are annotated with `@ConfigSecuritySensitive`

Reply:
Makes sense!!